openmdao_book/features/core_features/running_your_models/run_model.ipynb

```diff
@@ -0,0 +1,150 @@
+{
```

Comment:
<p>Split these to match http://openmdao.org/twodocs/versions/latest/features/core_features/running/run_model.html</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMDAO/OpenMDAO_Book/pull/77/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>